apps/web/app/lib/questions.tsx

```diff
@@ -230,3 +231,35 @@ export const getTSurveyQuestionTypeName = (id: string) => {
   const questionType = questionTypes.find((questionType) => questionType.id === id);
   return questionType?.label;
 };
+
+export const QUESTIONS_ICON_MAP: Record<TSurveyQuestionType, JSX.Element> = {
```

Comment:
Can we somehow avoid to declare things twice? We already listed out all the question types toegther with their icons in the `questionTypes` variable. could we reuse this instead of defining again what question types exists and what their icon and name is?